errtracker/errtracker.go

```diff
@@ -210,15 +315,28 @@ func report(errMsg, dupSig string, extra map[string]string) (string, error) {
 		"ErrorMessage":       errMsg,
 		"DuplicateSignature": dupSig,
 
+		"JournalError":       journalError(),
+		"ExecutablePath":     procExe(),
+		"ProcCmdline":        procCmdline(),
+		"ProcCpuinfoMinimal": procCpuinfoMinimal(),
+		"ProcCwd":            procCwd(),
+		"ProcEnviron":        environ(),
+		"DetectedVirt":       detectVirt(),
+		"SourcePackage":      "snapd",
+
 		"DidSnapdReExec": didSnapdReExec(),
 	}
+
+	if desktop, ok := osLookupEnv("XDG_CURRENT_DESKTOP"); ok {
```

Comment:
Hm, this will not ever work, right? I mean, snapd almost never runs inside xdg_current_desktop (unless run manually for development maybe)